### DISCUSSION TITLE
Neovide does not seem to work with ALE

### DISCUSSION THREAD

**Discussion by N8Brooks** [neovide/neovide#2269] [General]
If it isn't clear from my description, I was a vscode user and am new to vim, neovim, neovide etc.

**Describe the bug**
With nvim, ale linters show a red ">>" for lines that contain linting errors. With neovide, ale does not show that. 

**To Reproduce**

1. Open JavaScript file.
2. Write some JavaScript that contains a lint error.

**Expected behavior**
Ale linters should show a red ">>" for lines that contain linting errors on Neovide.

**Screenshots**

Neovide (actual):
![Screen Shot 2022-08-26 at 11 42 20 AM](https://user-images.githubusercontent.com/20604441/186970978-a83c4f83-05ad-4f66-8422-26053c95b29a.png)

Nvim (expected):
![Screen Shot 2022-08-26 at 11 38 38 AM](https://user-images.githubusercontent.com/20604441/186970395-d4a6f1da-61d2-48af-bb84-1dfd3690299a.png)

**Desktop (please complete the following information):**
 - OS: macOS Montery 12.5.1
 - Neovide Version 0.10.1
 - Neovim Version 0.7.2

**Please run `neovide --log` and paste the contents of the `.log` file created in the current directory here:**
Running `neovide --log` just opened a new neovide window for me?

**Additional context**
* I'm running neovide across ssh with the 6666 port and `--remote-tcp`
* I've installed neovim with AstroVim
* I've installed ale with these steps from the readme: https://github.com/dense-analysis/ale#neovim-on-unix
  * Using flow-language-server and eslint linters
  * Using prettier formatter
* I'm using the 'latte' flavour from the catppuccin theme.

**Comment by fredizzimo** [comment]
This is a configuration issue, not a Neovide bug, so converting to a discussion
